pyndl/ndl_parallel.pyx

```diff
@@ -52,29 +52,39 @@ def learn_inplace(binary_file_paths, np.ndarray[dtype_t, ndim=2] weights,
     cdef unsigned int length_all_outcomes = all_outcomes.shape[0]
     cdef char* fname
     cdef unsigned int start_val, end_val, ii, number_parts
-    cdef int error = 4
+    cdef int error = 3
+    # error codes:
+    #  0: no error
+    #  1: magic number does not match
+    #  2: version number does not match
+    #  3: error is never written
```

Comment:
In 'never written' the same as 'undefined', meaning: There was an error, but we don't have any details.

Reply:
It is the initialized value for error. Therefore no more information is available except, for that the variable was initialized with 3 and then returned.

Reply:
Do you have any idea how to phrase this better? Is there a standard way of doing this? I did not want to initialize the error variable with 0, therefore I picked a random non-zero number.

Reply:
Should be fine the way it is here. Readability / naming will improve by PR #173